core/build/webpack.base.config.ts

```diff
@@ -110,26 +121,45 @@ export default {
   resolveLoader: {
     modules: [
       'node_modules',
-      path.resolve(__dirname, themesRoot)
+      path.resolve(__dirname, themesRoot),
+      path.resolve(__dirname, moduleRoot)
     ]
   },
   resolve: {
     modules: [
       'node_modules',
-      path.resolve(__dirname, themesRoot)
+      path.resolve(__dirname, themesRoot),
+      path.resolve(__dirname, moduleRoot)
     ],
     extensions: ['.js', '.vue', '.gql', '.graphqls', '.ts'],
     alias: {
       // Main aliases
       'config': path.resolve(__dirname, './config.json'),
-      'src': path.resolve(__dirname, '../../src'),
+      'src': path.resolve(__dirname, config.projectRoot + '/src'),
```

Comment:
@yanko-ivanov why not a single alias to `@vue-storefront/core' ?

Reply:
Great question and one I've thought about much. For some reason, that didn't work in my experience. As you know, the @vue-storefront/core file path should be picked up from the package.json onwards, as it's in the node_modules. But for some reason deep linking doesn't work, and you have to explicitly resolve it. Prime examples are - as seen below - lib/ and lib/modules, data-resolver and data-resolver/types, and i18n as a whole. 
I know that this leaves us in a situation where the base webpack config is to be tracked and updated whenever the file structure changes, which isn't ideal, but I couldn't find a better solution.